app/tests/test_views.py

```diff
@@ -56,6 +56,34 @@ def login(self, email=None, password=None):
         assert LOGGED_IN_SENTINEL in res.data
         return res
 
+class ActivityFeedTests(ViewTestCase):
+    def test_posting_activity_requires_full_name(self):
```

Comment:
I don't think we need to test for this.  We shouldn't be letting users in without names, and if somehow one does slip in, I think we should just hide their post.